src/components/primitive/badge/badge.component.styles.ts

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { Box, Text } from 'components/primitive';
+
+/**
+ * Base styles for Badge Component.
+ */
+export const BaseBadge = styled(Box)`
+  height: 3.3rem !important;
+  width: fit-content;
+  padding: 1.2rem 0.6rem !important;
+  min-width: 10.5rem !important;
+  border-radius: 0.4rem;
+  border: none;
+  align-content: center !important;
+`;
+
+/**
+ * Style for SuccessBadge.
+ */
+export const SuccessBadge = styled(BaseBadge)`
+  background-color: ${({ theme: { colors } }) => colors.successLighter} !important;
+`;
+
+/**
+ * Style for DangerBadge.
+ */
+export const DangerBadge = styled(BaseBadge)`
+  background-color: ${({ theme: { colors } }) => colors.errorLighter} !important;
+`;
+
+/**
+ * Default styles for BadgeText.
+ */
+export const BadgeText = styled(Text)`
+  color: ${({ theme: { colors } }) => colors.white};
```

Comment:
The badge text color should be `success` or `error`

Reply:
success is the default variant so replaced the white color with SuccessLight.

Reply:
No. This has to be conditional, right?

Reply:
nope, this is a generic component. we can pass 2 props like success or error. based on states like pending, claimed conditional goes inside the screen.